CWE.Data/Models/Mute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CWE.Data.Models
+{
+    /// <summary>
+    /// Represents a mute structure to keep track of muted users.
+    /// </summary>
+    [Keyless]
+    public class Mute
+    {
+        /// <summary>
+        /// Gets or sets the infractio id for this mute.
+        /// </summary>
+        public Guid InfractionId { get; set; }
```

Comment:
Is there any reason this is called an `InfractionId`? I don't see a relationship to the infractions table, shouldn't this just be an id or `MuteId`?